the-rules/data/no-rules.json5

```diff
@@ -7,5 +7,13 @@
   // You may want to know why someone needs branches if there are no rules, well its a story about time
   // if I had more time I would explain it, but I'm still working on the rules of math, which are not so much rules as ...
   // sorry that may have been a rant there, keep going, and just define your own truth.
+  ruleOne: {
+    active: true,
+    parameters : [
+     "many different string statements allowed",
+     "we can propose any type of rule, like a rule to allow rules into the repo based on consensus",
```

Comment:
we could actually setup the cli to be able to route PR's. I think projects that have lots of PR's struggle with answering them, and part of being agile is getting the Pr's merged with the product owner. In this game I am both the product owner, and maintainer. However, if the game is fun it doesn't need to stay that way..